message_adapter/message_adapter.py

```diff
@@ -154,6 +155,13 @@ def __loadConfig(self, event, context):
 
         raise LookupError('Unknown event source: ' + source)
 
+    def __validate_json(self, document, schema_filepath):
```

Comment:
Is it possible to preface the error messages with which schema is failing?

Reply:
Good idea